python/mujincontrollerclient/controllergraphclient.py

```diff
@@ -0,0 +1,1595 @@
+            fields (list or dict): Optionally specify a subset of fields to return.
+            timeout (float): Number of seconds to wait for response.
+
+        Returns:
+            IKParameterization: Inverse Kinematics Parameter describe a pose in space with additional parameters that can affect grasping.
+        """
+        parameterNameTypeValues = [
+            ('bodyId', 'String!', bodyId),
+            ('environmentId', 'String!', environmentId),
+            ('ikParamId', 'String!', ikParamId),
+        ]
+        queryFields = {'angle': None, 'customData': {'deleted': None, 'id': None, 'values': None}, 'deleted': None, 'direction': None, 'id': None, 'localTranslate': None, 'name': None, 'quaternion': None, 'transform': None, 'translate': None, 'type': None}
+        return self._CallSimpleGraphAPI('query', operationName='GetIKParameterization', parameterNameTypeValues=parameterNameTypeValues, queryFields=queryFields, fields=fields, timeout=timeout)
+
+    def GetJoint(self, bodyId, environmentId, jointId, fields=None, timeout=None):
+        """Get a particular joint in a body.
+
+        Args:
+            bodyId (String!): ID of the body.
+            environmentId (String!): ID of the environment.
+            jointId (String!): ID of the existing joint.
+            fields (list or dict): Optionally specify a subset of fields to return.
+            timeout (float): Number of seconds to wait for response.
+
+        Returns:
+            Joint: Joint in a body describing the linkage between a parent link and a child link.
+        """
+        parameterNameTypeValues = [
+            ('bodyId', 'String!', bodyId),
+            ('environmentId', 'String!', environmentId),
+            ('jointId', 'String!', jointId),
+        ]
+        queryFields = {'anchors': None, 'axes': None, 'childLinkName': None, 'currentValues': None, 'deleted': None, 'electricMotorActuator': {'assignedPowerRating': None, 'coloumbFriction': None, 'gearRatio': None, 'maxInstantaneousTorque': None, 'maxSpeed': None, 'maxSpeedTorquePoints': None, 'modelType': None, 'noLoadSpeed': None, 'nominalSpeedTorquePoints': None, 'nominalTorque': None, 'nominalVoltage': None, 'rotorInertia': None, 'speedConstant': None, 'stallTorque': None, 'startingCurrent': None, 'terminalResistance': None, 'torqueConstant': None, 'viscousFriction': None}, 'floatParameters': {'deleted': None, 'id': None, 'values': None}, 'hardMaxAccel': None, 'hardMaxJerk': None, 'hardMaxVel': None, 'id': None, 'intParameters': {'deleted': None, 'id': None, 'values': None}, 'isActive': None, 'isCircular': None, 'lowerLimit': None, 'maxAccel': None, 'maxInertia': None, 'maxJerk': None, 'maxTorque': None, 'maxVel': None, 'mimics': {'equations': None}, 'name': None, 'offsets': None, 'parentLinkName': None, 'resolutions': None, 'stringParameters': {'deleted': None, 'id': None, 'value': None}, 'type': None, 'upperLimit': None, 'weights': None}
```

Comment:
@ziyan Do you need all these fields set to `None`?

Reply:
@rdiankov This describes a tree, such as `{ a, b, c { c1, c2, c3 } }`

We don't necessary need to use `None`, but we need something here to make python happy syntax wise.